packages/core/src/env-set/oidc.ts

```diff
@@ -5,6 +5,12 @@ import { getEnv } from '@silverhand/essentials';
 import inquirer from 'inquirer';
 
 const readPrivateKey = async (path: string): Promise<string> => {
+  const privateKey = getEnv('OIDC_PRIVATE_KEY');
```

Comment:
nit: feels like we could leave some comments here indicating bot from .env and .pem works the same.  Just for future reference.

Reply:
Agreed. Will add before merging